cardano-cli/golden/files/golden/help/shelley_governance_action_create-protocol-parameter-update.cli

```diff
@@ -0,0 +1,108 @@
+Usage: cardano-cli shelley governance action create-protocol-parameter-update --epoch EPOCH
+            --drep-cold-verification-key-file FILE
+            [--protocol-major-version NATURAL --protocol-minor-version NATURAL]
+            [--decentralization-parameter RATIONAL]
+            [--extra-entropy HEX | --reset-extra-entropy]
+            [--max-block-header-size NATURAL]
+            [--max-block-body-size NATURAL]
+            [--max-tx-size NATURAL]
+            [--min-fee-constant LOVELACE]
+            [--min-fee-linear LOVELACE]
+            [--min-utxo-value NATURAL]
+            [--key-reg-deposit-amt NATURAL]
+            [--pool-reg-deposit NATURAL]
+            [--min-pool-cost NATURAL]
+            [--pool-retirement-epoch-boundary EPOCH_BOUNDARY]
+            [--number-of-pools NATURAL]
+            [--pool-influence RATIONAL]
+            [--monetary-expansion RATIONAL]
+            [--treasury-expansion RATIONAL]
+            [--utxo-cost-per-word LOVELACE]
+            [--price-execution-steps RATIONAL --price-execution-memory RATIONAL]
+            [--max-tx-execution-units (INT, INT)]
+            [--max-block-execution-units (INT, INT)]
+            [--max-value-size INT]
+            [--collateral-percent INT]
+            [--max-collateral-inputs INT]
+            [--utxo-cost-per-byte LOVELACE]
+            [--cost-model-file FILE]
+            --out-file FILE
+
+  Create an protocol-parameter-update action
+
+Available options:
+  --epoch EPOCH            The epoch number in which the update proposal is
```

Comment:
We don't need --epoch; the user will not set an epoch, they just throw it out and it lands when it lands. There will be a protocol parameter that deals with deadlines.